client/src/components/RelatedItems/RelatedProducts/RelatedProducts.jsx

```diff
@@ -104,93 +102,147 @@ const RelatedProducts = ( { productId, setProductId } ) => {
   const onProductClick = (relatedProductId) => {
     setProductId(relatedProductId);
   };
+  // Carousel style
+  const carouselContainerStyle = {
+    position: 'relative',
+    width: '100%',
+    overflow: 'hidden',
+  };
+  
+  const carouselWrapperStyle = {
+    display: 'flex',
+    transform: `translateX(-${currentSlide * 30}%)`, 
+    transition: 'transform 0.3s ease-in-out',
+  };
+  
+  const carouselSlideStyle = {
+    flex: '0 0 30%',
+    opacity: 1,
+    transition: 'opacity 0.3s ease-in-out',
+    margin: '0 5px',
+  };
+
+  const carouselSlideActiveStyle = {
+    ...carouselSlideStyle,
+    opacity: 1,
+  };
+
+  const carouselControlStyle = {
+    position: 'absolute',
+    top: '50%',
+    transform: 'translateY(-50%)',
+    backgroundColor: '#fff',
+    border: 'none',
+    cursor: 'pointer',
+  };
+
+  const carouselPrevStyle = {
+    ...carouselControlStyle,
+    left: '10px',
+  };
+
+  const carouselNextStyle = {
+    ...carouselControlStyle,
+    right: '10px',
+  };
+  const handlePrev = () => {
+    setCurrentSlide(currentSlide > 0 ? currentSlide - 1 : Math.ceil(slides.length / 3.5) - 1);
+  };
+  
+  const handleNext = () => {
+      setCurrentSlide(currentSlide < Math.ceil(slides.length / 3.5) - 1 ? currentSlide + 1 : 0);
+  };
+  
+  const modalStyle = {
+    position: 'fixed',
+    top: '50%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+    backgroundColor: 'white',
+    padding: '20px',
+    zIndex: 1000,
+    display: isModalOpen ? 'block' : 'none',
+  };
+
+  const backdropStyle = {
+    position: 'fixed',
+    top: 0,
+    left: 0,
+    right: 0,
+    bottom: 0,
+    backgroundColor: 'rgba(0, 0, 0, 0.5)',
+    zIndex: 999,
+    display: isModalOpen ? 'block' : 'none',
+  };
+  const slideImageStyle = {
+    height: '380px',
+    width: '250px',
+    border: 'solid',
+    display: 'flex',
+    flexDirection: 'column',
+    position: 'relative', 
+    }
+  const imageStyle = {
+    height: '250px',
+    backgroundColor: 'grey',
+    position: 'relative',
+    overflow: 'hidden',
+    backgroundSize: 'cover',
+    backgroundRepeat: 'no-repeat',
+  };
+  const starStyles = {
+    position: 'absolute',
+    top: '10px',
+    right: '10px',
+    cursor: 'pointer',
+    color: 'green',   
+    zIndex: 10,
+  }
+  const relatedProductsContainerStyle = {
+    maxWidth: '80%', 
+    margin: '0 auto', 
```

Comment:
Change this to `margin: '0 6rem'` and remove the maxWidth property to conform with the rest of the page.